github.go

```diff
@@ -69,27 +69,53 @@ func checkOpenPRs(ctx *context.Context, client *github.Client, owner string, rep
 	}
 
 	for _, openPullRequest := range openPullRequests {
+		initialDate, err := initialDate(ctx, client, owner, repo, openPullRequest)
+		if err != nil {
+			fmt.Printf("Error %s\n", err)
+			os.Exit(1)
+		}
+
 		currentPr := prType{
 			client:    client,
 			ctx:       ctx,
 			owner:     owner,
 			repo:      repo,
-			OpenSince: daysSincePRCreated(openPullRequest.CreatedAt),
+			daysSince: daysSince(initialDate),
 			GHpr:      openPullRequest,
 		}
-		fmt.Printf("Repo %s/%s/#%d user '%s' open since '%d' days : ", owner, repo, *openPullRequest.Number, *openPullRequest.User.Login, currentPr.OpenSince)
+		fmt.Printf("Repo %s/%s/#%d user '%s' open since '%d' days : ", owner, repo, *openPullRequest.Number, *openPullRequest.User.Login, currentPr.daysSince)
 		// take action if any
 		actions(currentPr)
 	}
 }
 
-func daysSincePRCreated(CreatedAt *time.Time) int {
-	duration := time.Since(*CreatedAt)
+func initialDate(ctx *context.Context, client *github.Client, owner string, repo string, pullRequest *github.PullRequest) (*time.Time, error) {
```

Comment:
maybe lastActionDate or something like that initialDate makes no real sense for a last commit date right?